src/Icons/doc/index.rst

```diff
@@ -546,7 +549,7 @@ Full Configuration
 
         # Default attributes to add to all icons
         default_icon_attributes:
-            fill: currentColor
+            style: 'color: white;'
```

Comment:
Let's keep the documentation  as it was here (`fill: currentColor` is a default configuration value of this package)